### PR TITLE
Documentation created

### PR DESCRIPTION
**Pull request by northys**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+# doctrine-sortable
+
+Sooner or later you will have to implement sorting of your entities.
+For example categories, products on main page and so on. And why you should do this by yourself when everything you have to do is to copy & paste?
+
+
+Installation
+-----------
+
+The best way to install Kdyby/Doctrine is using [Composer](http://getcomposer.org/):
```
Comment (matej21) [inline README.md]: really? :D

File: README.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+```sh
+$ composer require librette/doctrine-sortable
+```
+
+and enable librette extension in your `config.neon
+````yml
+extensions:
+	# add this line at the end of your extensions list
+	librette.doctrine.sortable: Librette\Doctrine\Sortable\DI;
```
Comment (matej21) [inline README.md]: really? :D

Comment (enumag) [inline README.md]: `\SortableExtension` and no `;`

File: README.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+	 * @ORM\Id
+	 * @ORM\Column(type="integer")
+	 * @ORM\GeneratedValue
+	 */
+	protected $id;
+
+	/**
+	 * @ORM\Column(type="integer")
+	 */
+	protected $position;
```
Comment (matej21) [inline README.md]: both this property and getPosition method are useless, it is already in the trait

File: README.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+There is trait `TSortable` that implements basic sorting methods to your entity.
+Everything you need is to call those methods in your service / presenter.
+
+```php
+// you can move your entity up or down
+$entity->moveUp();
+$entity->moveDown();
+// or you can put it before / after another one
+$entity->moveBefore($anotherEntity);
+$entity->moveAfter($anotherEntity);
```
Comment (matej21) [inline README.md:61]: can you also mention setPosition method? thx

File: README.md (hunk 1 above)
Comment (enumag) [inline README.md]: @matej21 What? He's correct! :-D

Comment (northys) [inline README.md]: @enumag @matej21 anyone has something against documentation/sandbox? :-D

**Pull request closed without merging**